cmd/plugins/balloons/policy/balloons-policy.go

```diff
@@ -35,11 +35,12 @@ import (
 )
 
 const (
+	highPrio = cpuallocator.PriorityHigh
+	lowPrio  = cpuallocator.PriorityLow
```

Comment:
It looks to me as the original const block was deliberately 'exported consts first, linebreak, unexported consts'. @askervin Was it so, or did it just happen to look like that ? If it was so, I'd propose to reorder these additions to keep the original idea intact, IOW move these to the original 2nd block and reinsert the removed empty line.